codec/encoder/core/inc/param_svc.h

```diff
@@ -426,6 +432,11 @@ typedef struct TagWelsSvcCodingParam: SEncParamExt {
       pSpatialLayer->uiTransferCharacteristics = pCodingParam.sSpatialLayers[iIdxSpatial].uiTransferCharacteristics;
       pSpatialLayer->uiColorMatrix =             pCodingParam.sSpatialLayers[iIdxSpatial].uiColorMatrix;
 
+      pSpatialLayer->bAspectRatioPresent = pCodingParam.sSpatialLayers[iIdxSpatial].bAspectRatioPresent;
+      pSpatialLayer->eAspectRatio = pCodingParam.sSpatialLayers[iIdxSpatial].eAspectRatio;
+      pSpatialLayer->sAspectRatioExtWidth = BIG_ENDIAN_16(pCodingParam.sSpatialLayers[iIdxSpatial].sAspectRatioExtWidth);
```

Comment:
If you can build and run openh264 on big endian system, you can check if ffmpeg on little endian system can get correct "sar" values from the sps/vui , encoded with openh264 on big endian system. The values should be wrong..